source/includes/_visitor_events.md

```diff
@@ -8,11 +8,9 @@
 |---|---|
 |GET|/client_reports/visitor_events?[query_params]|
 
-This API can be used to retrieve capture events for a list of visitor_ids for a gmaid.
+This API should be used to retrieve all events for a visitor_ids for a given advertiser (GMAID). The response will contain a chornological list of events. It can also be optionally grouped by visitor_id. You can also request for events within a data range.
```

Comment:
"a list of visitor_ids"

Reply:
Last sentence is not grammatically correct.  Suggest: "Events can also be restricted to a date range".